packages/devtools_app/lib/src/shared/feature_flags.dart

```diff
@@ -59,8 +59,13 @@ abstract class FeatureFlags {
   /// Flag to enable offline data on memory screen.
   ///
   /// https://github.com/flutter/devtools/issues/5606
-  static const memoryOffline =
-      _kMemoryOfflineExperiment; // requires special handling because it needs to be const
+  static const memoryOffline = _kMemoryOfflineExperiment;
+
+  /// Flag to enable offline data on memory screen.
+  ///
+  /// https://github.com/flutter/devtools/issues/5606
+  static bool memorySaveLoad =
+      enableExperiments; // requires special handling because it needs to be const
```

Comment:
why are we creating another flag?

Reply:
because we agreed to remove save/load button while performance issue is not resolved.

So, memoryOffline is on and memorySaveLoad is off in prod

Reply:
can you differentiate the dart doc and tracking issue then if these are now different features?

Reply:
Done. Thanks for catch.